ansible/group_vars/database-hosts.yml

```diff
@@ -29,3 +33,18 @@ idr_omero_readonly_database:
   user: omeroreadonly
   password: "{{ idr_secret_postgresql_password_ro | default('omero') }}"
   statement_timeout: 60000
+
+postgresql_install_extensions: True
+
+# TODO: add a variable to easily enable/disable
```

Comment:
Is this worth capturing outside of idr/deployment?

Reply:
I see this as still in the experimental phase (which is why it's commented out), but if it turns out to be useful we might just enable it unconditionally.